packages/flutter_tools/lib/src/error_messages.dart

```diff
@@ -0,0 +1,177 @@
+import 'base/context.dart';
+import 'ios/cocoapods.dart';
+import 'vscode/vscode_validator.dart';
+
+ErrorMessages get errorMessages => context[ErrorMessages];
+
+/// Class containing the messages that can be produced by the doctor validators.
+/// Members are split by validator and are given in the order they appear in the
+/// validator code.
+class ErrorMessages {
```

Comment:
These are all the messages so let's just call them `Messages` or `Strings` since messages seems to conflict in some files.

Reply:
Or perhaps `UserMessages` (as opposed to bus messages, for instance)

Reply:
Named to UserMessages.